code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -49,7 +50,7 @@
 		return
 
 	to_chat(user, "<span class='notice'>You fold [src] flat.</span>")
-	var/obj/item/I = new foldable(get_turf(src))
+	var/obj/item/stack/I = new foldable(get_turf(src), amt)

```

Comment:
I'm not gonna ask you to change this but it's absolutely worth noting that this (`/item/I` -> `/item/stack/I`) doesn't matter in the slightest. All procs are called on the instance, not the path.